src/settings/MappingProfiles/MappingProfilesFormContainer/MappingProfileFormContainer.test.js

```diff
@@ -0,0 +1,214 @@
+    records: [],
+    outputFormat: 'MARC',
+  },
+  onSubmit = noop,
+}) => {
+  const intl = useIntl();
+
+  console.error = jest.fn();
+
+  return (
+    <SettingsComponentBuilder>
+      <MappingProfilesFormContainer
+        allTransformations={generateTransformationsWithDisplayName(intl, allTransformations)}
+        contentLabel={contentLabel}
+        isEditMode={isEditMode}
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+      />
+    </SettingsComponentBuilder>
+  );
+};
+
+describe('MappingProfileFormContainer', () => {
+  describe('rendering MappingProfileForm', () => {
+    beforeEach(() => {
+      renderWithIntl(
+        <MappingProfileForm
+          allTransformations={allMappingProfilesTransformations}
+        />,
+        translationsProperties,
+      );
+    });
+
+    it('should place accordion headers in correct order', () => {
+      const headers = document.querySelectorAll('[data-test-headline]');
+
+      expect(getByText(headers[0], 'Summary')).toBeVisible();
+      expect(getByText(headers[1], 'Transformations')).toBeVisible();
+    });
+
+    it('should place Summary fields in correct order', () => {
+      const labels = document.querySelectorAll('[data-test-accordion-section] label');
+
+      expect(getByText(labels[0], 'Name')).toBeVisible();
+      expect(getByText(labels[1], 'Output format')).toBeVisible();
+      expect(getByText(labels[2], 'FOLIO record type')).toBeVisible();
+      expect(getByText(labels[3], 'Source record storage (entire record)')).toBeVisible();
+      expect(getByText(labels[4], 'Inventory instance (selected fields)')).toBeVisible();
+      expect(getByText(labels[5], 'Holdings')).toBeVisible();
+      expect(getByText(labels[6], 'Item')).toBeVisible();
+      expect(getByText(labels[7], 'Description')).toBeVisible();
+    });
+
+    it('should disable Instance record type option when selecting SRS', () => {
+      userEvent.click(screen.getByRole('checkbox', { name: 'Source record storage (entire record)' }));
+
+      expect(screen.getByDisplayValue('INSTANCE')).toBeDisabled();
+    });
+
+    it('should disable Instance record type option in Transformation list when selecting SRS', () => {
+      userEvent.click(screen.getByRole('checkbox', { name: 'Source record storage (entire record)' }));
+      userEvent.click(screen.getByRole('button', { name: 'Add transformations' }));
+      const modalRecordFilters = screen.getByRole('region', { name: 'Record type filter list' });
+
+      expect(getByRole(modalRecordFilters, 'checkbox', { name: 'Instance' })).toBeDisabled();
+    });
+
+    it('should disable SRS record type option when selecting Instance', () => {
+      userEvent.click(screen.getByRole('checkbox', { name: 'Inventory instance (selected fields)' }));
+
+      expect(screen.getByRole('checkbox', { name: 'Source record storage (entire record)' })).toBeDisabled();
+    });
+
+    it('should close transformations modal when clicking Cancel', () => {
```

Comment:
Please use the specified wording for consistency.
```suggestion
    it('should close transformations modal when clicking on cancel button', () => {
```